app/Lib/Export/NidsExport.php

```diff
@@ -16,11 +16,14 @@ class NidsExport
 				'fields' => array('threat_level_id')
 			)
 		),
-		'flatten' => 1
+		#'flatten' => 1
 	);
 
 	public function handler($data, $options = array())
 	{
+	
+		//NOTES: Here the scope "Object" should be probably checked
```

Comment:
Not sure I understand the meaning of this comment. However, it should be noted that the `network-connection` object is only supported when the scope `event` is used. Would it be worth investing more time trying to have the same feature while using the `attribute` scope?

Reply:
The comment should have been deleted as it was written at the very beginning while studying the code. However, I am not sure I get exactly the situation in which an "object" signature will be created while using the "attribute" scope. Could you make an example?